docs/index.md

```diff
@@ -1,241 +1,172 @@
----
+<!-- ---
 hide:
 - navigation
----
+--- -->
 
-# Introducing Pydantic Logfire
+# Get Started
```

Comment:
since this is the very start of the index page of our docs, I think we need one paragraph explaining what logfire is and/or selling it a little bit. (a bit like the first two sentences [here](https://docs.pydantic.dev/latest/))

It can be one or two sentences, and I guess should include a link to pydantic.dev/logfire.

Reply:
I also these we need one sentence before the steps start, so overall something like this:

---

# Pydantic Logfire (h1)

From the team behind Pydantic, Logfire is a new type of observability platform built on the same belief as our open source library — that the most powerful tools can be easy to use.

Logfire is built on OpenTelemetry, and supports monitoring your application from any language, but we have particularly great support for Python!

_(Or something)_

## Getting Started (h2)

Below is a quick walk through of how you can start using Logfire with a Python app.

### Logfire Platform (h3)

In the logfire platform:

1. [Log into Logfire](#)
2. Follow the prompts to create your account
3. From your Organisation, click "New project" and create your first project
4. ...

### Install SDK

...

Reply:
This is done, I linked to the `Why Logfire` section to keep you within the docs, this made the most sense to me.